### ISSUE TITLE
Separate "Earn more PTI" in digestable steps

### ISSUE THREAD

**Issue by felipegaucho** [Paratii-Video/paratii-design#2]
**Current situation**: "Earn more PTI" returns a modal with 3 "pages", each with a list of reward/action pairs. This can be more user friendly if actions are presented progressively and each one gives a little (different?) push for the user to complete it.

**Proposal 1**: to break down the list in an ordered and user-friendly process.

**References**:
- [21.co does it well and simply](https://imgur.com/a/Mwc7a) (try signing up and earning a dollar by verifying email, etc). Then check your ["history"](https://imgur.com/a/7uqjt).
- [Paypal is a bit more charmy](https://zapier.cachefly.net/storage/photos/74284e890825e2a655192e8048ecd0dd.png).

**Proposal 1.b** - Also, it'd be great if the user gets in process already with a "headstart" (1st step of 10 already done - to registrate). 

_(theoretical)_ **References**: Like in the ["customer loyalty headstart experiment"](https://stampme.com/accelerate-your-loyalty-program-give-your-customers-a-head-start/). [This paper](https://msbfile03.usc.edu/digitalmeasures/jnunes/intellcont/Endowed%20Progress%20Effect-1.pdf) makes a good point for it.

**Comment by pedrocasa** [comment]
Just to make clear to everyone before those changes are made.

( those are the earn more PTI pages )
![m28](https://user-images.githubusercontent.com/30123644/31611719-fbe17484-b25b-11e7-8f71-89568d47400e.png)
![m29](https://user-images.githubusercontent.com/30123644/31611720-fc2c7722-b25b-11e7-9a33-c13ffdc3b68b.png)
![m27](https://user-images.githubusercontent.com/30123644/31611721-fc47900c-b25b-11e7-9e5a-59f7908ff73d.png)

**Comment by jellegerbrandy** [comment]
I think these kind of list should not be in the player. 
Ideally, they are a part of a whole "documentation suite"- i.e. a average-user-understandable text in which we explain the paratii vision, talk about p2p stuff, etc etc. This should be on http://paratii.video. 

Also, most of the items on the list will be implemented later, and so adding this copy to the player now is mostly confusing/misleading. 

Personally, I think that in the player, these items should be "discoverable". For example, once we have a "flag" button, we can add a pop-up to that button which explains the function of that button and the associated rewards.

**Comment by felipegaucho** [comment]
Right, Jelle, with scope clearer, let's leave "earn more PTI" out of range for now.
For next iterations, I agree this should all be discoverable, and like the pop-ups/hovers. Ordering items on a flow (like Skype example) seems useful to me, too, on an isolated place in the app (i.e. to keep track of your "progress").
For now, agreeing w/ you, this should be on our website only =)
